cmd/bosun/expr/expr.go

```diff
@@ -152,6 +152,30 @@ func (s Series) MarshalJSON() ([]byte, error) {
 	return json.Marshal(r)
 }
 
+// sometimes, we prefer not to have the Series in its map format
+// but rather as a sorted array
+// see discussion at https://github.com/bosun-monitor/bosun/pull/699
+
+type sortablePoint struct {
+	t time.Time
+	v float64
+}
+
+type sortableSeries []sortablePoint
+
+func (s sortableSeries) Len() int           { return len(s) }
+func (s sortableSeries) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
+func (s sortableSeries) Less(i, j int) bool { return s[i].t.Before(s[j].t) }
+
+func NewSortedSeries(dps Series) sortableSeries {
```

Comment:
hmm but since Series is exported, it probably makes sense to export this functionality as well (i.e. make it `SortableSeries`)